deployment/check.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+---
+- name: Check a deployment
+  hosts: all
+  vars:
+    project_dir: "{{ playbook_dir }}/.."
+    deployment: "{{ lookup('env', 'DEPLOYMENT') }}"
+    DCs:
+      - redis
+  # For yet unknown reasons the dist2src-worker-0 pod claims to be Pending even we waited
+  # for it to be running in roles/deploy/tasks/wait-for-pods.yml
+  # Investigating the logs it seems that something actually spawns another worker pod between
+  # the wait-for-pods and this check because they have different timestamps and the first one has
+  # "image": "172.30.1.1:5000/myproject/dist2src-worker:latest"
+  # while the checking one has
+  # "image": "docker.io/usercont/dist2src-worker@sha256:9d8fc9de8c..."
+  # TODO: figure out why/what/where and uncomment this
+  #      - dist2src-worker
```

Comment:
Seems like an image lookup issue, though the image stream is correctly configured :thinking:

Reply:
Wild guess: maybe the issue is that when the image stream is created, there is no wait to the image to be actually pulled in the internal registry. So when the STS is being deployed, it won't find the image stream, thus pulls from docker.io. When pulling the image for the image stream finishes, a new STS rollout is triggered, b/c we have it configured.

Reply:
> Did the last success happen only by chance?

It succeeded because I commented out the checking of dist2src-worker in the last commit.

> maybe the issue is that  ...

We [push the image into internal registry manually](https://github.com/packit/dist-git-to-source-git/blob/master/deployment/roles/deploy/tasks/push-dev-images.yml) and the first pod is created from that image (`172.30.1.1:5000/myproject/dist2src-worker:latest`), but then probably another image (the `docker.io/usercont/dist2src-worker@sha256:9d8fc9de8c...` ) is imported into the image stream and a new STS rollout is triggered.
No idea what we do differently here than in `deployment` repo, where it works.

Reply:
Maybe turn off `importPolicy.scheduled` in the image stream?

Reply:
But we want the images to be imported from docker hub automatically, don't we?

Reply:
Ok, that was the problem. Seems to be fixed now. PTAL